docs/pages/enroll-resources/application-access/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Okta apps and groups sync
```

Comment:
We use title case for titles. I would give the [Style guide](https://www.notion.so/goteleport/Style-guide-160fdd3830be80a79e70f08df2df2a0a) a read when you get a chance.